mne/decoding/transformer.py

```diff
@@ -33,20 +33,37 @@ class Scaler(TransformerMixin):
     ----------
     info : instance of Info
         The measurement info
+    scaler : string, defaults to "standard"
+        if scaler == 'robust' sklearn RobustScaler() is used.
+        if scaler == 'standard' applies ``X/std-mean``
+        if scaler == 'default', scales mag by 1e15
+                                       grad by 1e13
+                                       eeg by 1e6
     ``ch_mean_`` : dict
         The mean value for each channel type
     ``std_`` : dict
         The standard deviation for each channel type
      """
-    def __init__(self, info, with_mean=True, with_std=True):
+    def __init__(self, info, scaler='standard', with_mean=True, with_std=True):
+        from sklearn.preprocessing import RobustScaler
+        from mne.io.pick import pick_info, _pick_data_channels, _picks_by_type
         self.info = info
         self.with_mean = with_mean
         self.with_std = with_std
-        self.ch_mean_ = dict()  # TODO rename attribute
-        self.std_ = dict()  # TODO rename attribute
+
+        if scaler not in ('robust', 'standard', 'default'):
+            raise ValueError("Invalild method for scaling")
+        
+        if scaler == 'robust':
+            scaler = RobustScaler()
```

Comment:
I'm not proposing `Scaler(method=sklearn.preprocessing.RobustScaler())`. I was asking why you can't pipeline RobustScaler after using EpochsVectorizer? What am I missing?